cohttp-eio/src/dune

```diff
@@ -1,4 +1,4 @@
 (library
  (name cohttp_eio)
  (public_name cohttp-eio)
- (libraries eio http fmt ptime))
+ (libraries cohttp eio eio.mock fmt http logs ptime uri uri.services))
```

Comment:
The main library shouldn't depend on `eio.mock` (which is for testing). It should go in `tests/dune` instead.